Sources/Widgets/Representations/WidgetRepresentation/index.js

```diff
@@ -173,6 +215,20 @@ const DEFAULT_VALUES = {
   actors: [],
   labels: [],
   behavior: Behavior.CONTEXT,
+  coincidentTopologyParameters: {
+    Point: {
+      factor: -1.0,
+      offset: -1.0,
+    },
+    Line: {
+      factor: -1.0,
+      offset: -1.0,
+    },
+    Polygon: {
+      factor: -1.0,
+      offset: -1.0,
+    },
+  },
```

Comment:
Why are those default values different here than from the widget manager example? Is one more recommended?

Reply:
The example values are different for an examples' sake, just so we don't set values to their defaults.

The default values should usually be fine.